addons/website/static/src/components/website_loader/website_loader.js

```diff
@@ -103,6 +110,47 @@ export class WebsiteLoader extends Component {
         });
     }
 
+    /**
+     * Initializes the progress bar.
+     */
+    initProgressBar() {
+        if (this.updateProgressInterval) {
+            return;
+        }
+        // The progress speed decreases as it approaches its limit. This way,
+        // users have the feeling that the website creation progressing is fast
+        // and we prevent them from leaving the page too early (because they
+        // already did XX% of the process).
+        // If there is no module to install, we fake the progress from 0 to 100.
+        // If there is at least 1 module to install, we take 70% of the progress
+        // bar that we divide by the number of modules to install. We fake the
+        // progress of each module individually and when all modules are
+        // installed, we fake the progress of the remaining 30%.
+        const nbModulesToInstall = this.featuresInstallInfo.total || 0;
+        const isSomethingToInstall = nbModulesToInstall > 0;
+        let currentProgress = 0;
+        // This controls the speed of the progress bar.
+        const progressStep = isSomethingToInstall ? 0.04 : 0.02;
+        let progressForAfterModules = isSomethingToInstall ? 30 : 100;
```

Comment:
Instead of the 30% being "after modules", can't it be "during the modules"? Meaning you would always have something moving. I wonder also if 30 is enough.
= instead of "currentProgress", you would have "currentModuleProgress" and "currentConfiguratorProgress" (something like that), going at the same time

Reply:
Nevermind, what I said does not change much (only when there are a lot of apps to install, in which case the 70 is divided a lot and it felt like it was not moving enough to me but it's good).